src/server/terrain/structure_building_blocks.zig

```diff
@@ -342,26 +342,25 @@ pub fn registerSBB(structures: *Assets.ZonHashMap) !void {
 	std.debug.assert(structureList.items.len == 0);
 	std.debug.assert(structureMap.capacity() == 0);
 
-	structureList.resize(main.worldArena, structures.count());
+	structureList.ensureCapacity(main.worldArena, structures.count());
 	structureMap.ensureTotalCapacity(main.worldArena.allocator, structures.count()) catch unreachable;
 
 	childrenToResolve = .init(main.stackAllocator);
 	defer childrenToResolve.deinit();
+	var loadedCount: u32 = 0;
```

Comment:
Why did you move this into the outer scope? Do you plan to use it somewhere below in the future?

Reply:
I did originally but I don't really need that anymore. I'll revert that change.